File: 283._Move_Zeroes/moveZeros.py
def moveZeros(nums):
    count = 0
    currentElement = 0;
    for i in range(len(nums)):
        if nums[i] != 0:
            nums[currentElement] = nums[i]
            count+= 1
            currentElement += 1
    for j in range(len(nums)):
        if j >= count:
         nums[j] = 0
    return nums

print(moveZeros([0, 1, 0, 3, 12]))

File: 283._Move_Zeroes/MoveZeros.java
import java.util.Arrays;

public class MoveZeros {
  int[] moveZeros(int[] nums) {
    int count = 0;
    int currentElement = 0;
    for (int i = 0; i < nums.length; i++) {
      if (nums[i] != 0) {
        nums[currentElement] = nums[i];
        currentElement++;
      } else {
        count++;
      }
    }
    for (int j = nums.length - count; j < nums.length; j++) {
      nums[j] = 0;
    }
    return nums;
  }

  public static void main(String[] args) {
    MoveZeros mz = new MoveZeros();
    int[] arr = { 0, 1, 0, 3, 12 };
    System.out.println(Arrays.toString(mz.moveZeros(arr)));
  }
}